recipe/meta.yaml

```diff
@@ -13,14 +13,26 @@ source:
 
 build:
   number: 0
-  script: pip install --no-deps .
+  script: python setup.py install --single-version-externally-managed --record record.txt
```

Comment:
Mike, I will get a Windows machine tomorrow and dig into this. Maybe conda-forge's pinnings are in play here.